code/parse/sexp.cpp

```diff
@@ -33959,7 +33964,22 @@ const char *sexp_error_message(int num)
 			return "Invalid traitor override";
 
 		case SEXP_CHECK_INVALID_LUA_GENERAL_ORDER:
-			return "Invalid lua general order";
+			return "Invalid Lua general order";
+
+		case SEXP_CHECK_INVALID_SHIP_POINT:
+			return "Invalid ship/waypoint name";
+
+		case SEXP_CHECK_INVALID_SHIP_WING_SHIPONTEAM_POINT:
+			return "Invalid ship/wing/ship on team/waypoint name";
+
+		case SEXP_CHECK_INVALID_SHIP_WING_POINT:
+			return "Invalid ship/wing/waypoint name";
+
+		case SEXP_CHECK_INVALID_ORDER_RECIPIENT:
+			return "Invalid order recipient";
+
+		case SEXP_CHECK_INVALID_SHIP_WING_WHOLETEAM:
+			return "Invalid ship/wing/team name";
```

Comment:
Looks like this is a good change, but these message are a little obtuse. (Yes, I know the format matches the old ones, but is that a good thing?)

Would it perhaps be clearer to say, at least for some of these messages something like, "Invalid name that does not match a ship, wing or team"?

Honestly without some clarification, I would not be able to tell what an "Invalid ship/wing/ship" is supposed to be.

Reply:
I could change this to "Invalid ship, wing, or team name", with similar changes to the other messages that contain forward slashes.  Would that work?

Reply:
I think so

Reply:
Ok, done